packages/clerk-js/src/core/clerk.ts

```diff
@@ -1556,9 +1555,7 @@ export default class Clerk implements ClerkInterface {
       false,
     );
 
-    return this.buildUrlWithAuth(
-      appendUrlsAsQueryParams(appendAsQueryParams(signInOrUpUrl, options?.initialValues || {}), opts),
-    );
+    return this.buildUrlWithAuth(appendAsQueryParams(signInOrUpUrl, { values: options?.initialValues, urls }));
```

Comment:
Passing `values` and `urls` as separate paramaters here is a bit confusing to me, particularly since they're handled with a single search params abstraction on the other end

Can we strip the origin from `urls` before calling `buildUrlWithAuth`, so that we only need to pass in a single object?

Reply:
Made a refactor commit.